go/vt/vtgr/controller/group.go

```diff
@@ -31,22 +33,31 @@ import (
 var (
 	groupOnlineSize = stats.NewGaugesWithMultiLabels("MysqlGroupOnlineSize", "Online MySQL server in the group", []string{"Keyspace", "Shard"})
 	isLostQuorum    = stats.NewGaugesWithMultiLabels("MysqlGroupLostQuorum", "If MySQL group lost quorum", []string{"Keyspace", "Shard"})
+
+	heartbeatThreshold = flag.Int("group_heartbeat_threshold", 0, "Group heartbeat staleness threshold. Need to set together with -enable_heartbeat_check")
```

Comment:
It will be good to write a bit more description for the flag. It's unclear what this flag really affects.

Reply:
Also is that supposed to be seconds? The unit is unclear too.

Reply:
Updated the description and added `_sec` suffix.